### ISSUE TITLE
Angular: logs page

### ISSUE THREAD

**Issue by pascalgrimaud** [jhipster/jhipster-lite#1653]
After generating Angular as client, it could be nice to have the possibility to generate logs page:

- entry in menu
- log html
- log component
- log service
- log tests

SImilar to:
![image](https://user-images.githubusercontent.com/9156882/167257843-53cca5ad-e132-4472-a2a1-717e48a47113.png)

**Comment by qmonmert** [comment]
@NicolasDesnoust yes it is possible :) it would be good to wait that the #1652 be merged because this PR lays the foundations

**Comment by NicolasDesnoust** [comment]
Indeed, I agree
